### ISSUE TITLE
Administration almost not usable after update (9.6)

### ISSUE THREAD

**Issue by SaliC-re** [WWBN/AVideo#3861]
Hey there,

i updated my AVideo from 9.4 to the newest 9.6 and after that I could almost not use administration.
After minutes and minutes of waiting and many retries i could update to 9.5 and 9.6.

Now all is up to date but all the site is very slow and loads at least 2-3 minutes after every click.
You can see the error logs here:
[https://pastebin.com/UviB96Zb](https://pastebin.com/UviB96Zb)

Hope you have some ideas

PS: Server is restarted and all caches are cleared

**Comment by DanielnetoDotCom** [comment]
I can see a lot of errors in your logs but I am not sure where they come from

You are right they looks like memory issues .

How is your hardware?

Can you please update again from today?

There is a more stable version on github

**Comment by SaliC-re** [comment]
Hey @DanielnetoDotCom 
did a upgrade some minutes ago
same again
without logging in the site is usable
after clicking enter to log in the site needs 2-3 minutes for every change of site

errors:
 [14-Sep-2020 23:06:44 Europe/Berlin] AVideoLog::SECURITY: ddosProtection:: progressive timeout timeoutReal = (5) active_connections = (40) maxCon = (40)  SCRIPT_NAME: /view/error.php

all logs:
[https://pastebin.com/h001pYhw](https://pastebin.com/h001pYhw)

**Comment by DanielnetoDotCom** [comment]
Ok, I still can see some errors.

why don't you have this file https://github.com/WWBN/AVideo/blob/master/plugin/CustomizeAdvanced/autoHideNavbar.js ?

`[19-Oct-2020 15:34:12 Europe/Berlin] PHP Warning:  file_get_contents(path/to/my/streamer/site/plugin/CustomizeAdvanced/autoHideNavbar.js): failed to open stream: No such file or directory in path/to/my/streamer/site/plugin/CustomizeAdvanced/CustomizeAdvanced.php on line 222`

Are you sure you update it correctly?

are you using `git pull `to update?

**Comment by SaliC-re** [comment]
hey,

I even did git reset --hard so I am fully up to date and the file is on place.
![image](https://user-images.githubusercontent.com/63057086/96584020-e4c32a00-12dd-11eb-8cb5-72f019343ac5.png)

**Comment by SaliC-re** [comment]
@DanielnetoDotCom 
I see that I posted too much logs for you. You only have to look for logs today because I did the update some minutes ago.
So just forget the old logs.

Can I clear the logs with GUI or DB?

**Comment by DanielnetoDotCom** [comment]
Yes, delete the log file and run again, then share it. it may help to identify what may be the issue.

also please check if you have premium plugins to update.

**Comment by SaliC-re** [comment]
i cleared everything (log, cache and so on) and still laggy.
only this warning:

![image](https://user-images.githubusercontent.com/63057086/96590413-47b8bf00-12e6-11eb-9b73-a22631d0e381.png)

**Comment by JoshWho** [comment]
Remove the PHP Memory limit in your .htaccess file in your website root

Mine did the same thing and was only trying to use 512mb when My site I know uses gigs of memory to function.

**Comment by SaliC-re** [comment]
@JoshWho 
nice trick! it's really nice and fast now. I had 512 M in the .htaccess and I changed it to 6G

@DanielnetoDotCom 
can you put 1-2G at default for the .htaccess?
I'm not good in this GIT game :)

**Comment by jengajenga** [comment]
Well, I seem to be having this problem as well. I cannot update my theme. I also set autoplay and it still needs me to press play on the website.
I have set memory limit to 2G. I have restarted apache2 but still the same. I am on the very very latest a1213ae..
@DanielnetoDotCom 
Thanks.

**Comment by DanielnetoDotCom** [comment]
@jengajenga did you clear the cache after change the theme?

**Comment by jengajenga** [comment]
> @jengajenga did you clear the cache after change the theme?

Yes. 
Cleared first page cache
Cleared cache directory
Cleared browser cache
Reloaded apache2
Use different browser
Use different laptop

I've noticed some themes are no longer working. Most light themes aren't working but the dark ones are.

**Comment by DanielnetoDotCom** [comment]
@jengajenga I notice you changed the code for the navbar and probably some other things as well.

I can only help you if you run the exact same code as me. I cannot help you if you changed it.

**Comment by SaliC-re** [comment]
Hi there again.
We have this problems with the long loading site again. 
I am on the newest version of avideo.
I saw that there is a function that checks the site for updates:
[18-Nov-2020 08:00:43 Africa/Abidjan] PHP Notice:  Undefined index: thereIsAnyRemoteUpdate in /var/www/html/AVideo/objects/functions.php on line 2280

We checked our company firewall and we saw that a request was consequently dropped. So we opened that port and the site was usable again. Until this every click needed 2 minutes - but only after login with an admin user. With version 9.4 this problems did not occur.
